easybuild/easyblocks/m/mxnet.py

```diff
@@ -97,21 +101,26 @@ def extract_step(self):
 
         for srcdir in [d for d in os.listdir(self.builddir) if d != os.path.basename(self.mxnet_src_dir)]:
             submodule, _, _ = srcdir.rpartition('-')
-            newdir = os.path.join(self.mxnet_src_dir, submodule)
+
+            # if newdir starts with 'oneDNN-', we rename it to mkldnn:
```

Comment:
Rather than commenting out what the code literally does, please explain why this is necessary

Reply:
solved